packages/revocation/src/implementations/statuslistentry-verification.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  CredentialSubject,
+  StatusList2021Entry,
+  VerifiableCredential,
+} from '@ew-did-registry/credentials-interface';
+import * as pako from 'pako';
+import axios from 'axios';
+
+/**
+ * A Class to verify the StatusList2021Entry
+ */
+export class StatusListEntryVerification {
```

Comment:
Could you document the expected API/usage of this class please? It seems to me like it would be as follows
```suggestion
/**
 * A Class to verify the StatusList2021Entry
 * Sample usage:
 *  const entryVerifier = new StatusListEntryVerification(vcProofVerificationCallback);
 *.  entryVerifier.verifyStatusList(statusListEntry);
 *.  const credential = entryVerifier.fetchCredential(statusListEntry.credentialStatus);
 *.  entryVerifier.verifyEncodedStatusList(credential);
 */
export class StatusListEntryVerification {
```
The above seems quite cumbersome, IMO. Can we simplify it to one API call? Just `entryVerifier.verifyStatusListEntry` that does everything?